src/pages/home/items-list.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const ItemList = [{image: 'https://cdn.shopify.com/s/files/1/1422/9358/products/BRITISH-DRUM-COMPANY-BDC-12X5.5-IMP-SNARE-DRUM-KENSINGTON-KNIGHT-DRUMAZON_01_1000x.jpg?v=1594060928', title: 'BDC Imp Snare Drum', price: '£379', location:'/item/1'},
```

Comment:
we should import this into the item context and use it as the initial value for items. That way we have easy access to it on other pages just by using the context